### PR TITLE
Prepare bump for `xchain-client|util`

### PR DESCRIPTION
**Pull request by veado**
- [x] `xchain-client@0.10.0`
- [x] `xchain-util@0.3.0`

### CODE REVIEW

File: packages/xchain-client/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# v.0.10.0 (2021-07-07)
+
+### Breaking change
+
+- Introduce `Network`, `TxType`, `FeeOption` enums
+- added support for fetch gas fees from thorchain
```
Comment (veado) [inline packages/xchain-client/CHANGELOG.md:6]: Remove that line ^

Comment (reidrankin) [inline packages/xchain-client/CHANGELOG.md]: this is already covered in the 0.9.4 entry
```suggestion
```

**Review by reidrankin — APPROVED** [review]
note the comment; otherwise, LGTM

File: packages/xchain-client/CHANGELOG.md (hunk 1 above)
Comment (reidrankin) [reply packages/xchain-client/CHANGELOG.md]: (lol i was too slow, this done be fixed already)

**Comment by reidrankin** [comment]
oh right you gotta bump the dep versions all around too I guess

**Comment by veado** [comment]
> oh right you gotta bump the `package.json` versions all around too I guess

It will be another (next) PR. `xchain-client|util` need to be published before, in other case all other packages will fail (as we currently have with `xchain-binance`)

**Comment by veado** [comment]
For now we ignore failing tests for `xchain-binance` (see https://github.com/xchainjs/xchainjs-lib/pull/388/checks#step:6:27) - it needs to be updated with latest `xchain-client|util`
```
Error: /home/runner/work/xchainjs-lib/xchainjs-lib/packages/xchain-binance/src/client.ts(18,3): semantic error TS2305: Module '"@xchainjs/xchain-client"' has no exported member 'singleFee'.
```

**Comment by reidrankin** [comment]
that's not it, it's that the deps don't match as specified any more

**Comment by veado** [comment]
> that's not it, it's that the deps don't match as specified any more

Yep, but you can't add dependencies of `xchain-client@0.10.0` / `xchain-util@0.3.0` to `xchain-binance` as long as these are not published or do I miss something?

**Comment by reidrankin** [comment]
I'm pretty sure you can -- I think it's failing because lerna is putting the `xchain-binance` build ahead of the `xchain-client` one, because it's noticed that `"@xchainjs/xchain-client": "^0.9.3"` dep in `xchain-binance` doesn't match the version in `/packages/xchain-client/package.json` so it hasn't realized they have to be built in order.

**Pull request merged**